gspread/worksheet.py

```diff
@@ -3336,3 +3338,57 @@ def add_validation(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def expand(
+        self,
+        top_left_range_name: str = "A1",
+        direction: TableDirection = TableDirection.table,
+        value_render_option: Optional[ValueRenderOption] = None,
+        date_time_render_option: Optional[DateTimeOption] = None,
```

Comment:
I think maybe we do not want to duplicate all the options for `get`. We would rather provide the user no choice and say

"if you want to get different values, use `get` as you desire, and then use `utils.expand_table`"

as there are already missing options (e.g., `combine_merged_cells`, `pad_values`), and more may be added in future

so we remove value_render_option and date_time_render_option in favour of telling user to use `get` themself. what do you think?